test/test_xslt_transforms.rb

```diff
@@ -268,7 +270,7 @@ def test_non_html_xslt_transform
         <xsl:output encoding="UTF-8" indent="yes" method="xml" />
 
         <xsl:template match="/">
-          <xsl:value-of select="/a" />
+          <a><xsl:value-of select="/a" /></a>
```

Comment:
@sparklemotion/nokogiri-core I'm not sure why libxml is not complaining about this transformation. the output is definitely not an xml document. instead of bending the java implementation to support libxml forgiveness, I changed the test to output proper xml. Any objections ?